doc/topics/windows/windows-package-manager.rst

```diff
@@ -334,200 +339,230 @@ you need to install a specific version of a piece of software.
-        uninstall_flags: '/qn /norestart'
-
-:param msiexec: This tells salt to use ``msiexec /i`` to install the
-    package and ``msiexec /x`` to uninstall. This is for `.msi` installations.
-    Possible options are: True, False or path to msiexec on your system
-
-    7zip:
-      '9.20.00.0':
-        installer: salt://win/repo/7zip/7z920-x64.msi
-        full_name: 7-Zip 9.20 (x64 edition)
-        reboot: False
-        install_flags: '/qn /norestart'
-        msiexec: 'C:\Windows\System32\msiexec.exe'
-        uninstaller: salt://win/repo/7zip/7z920-x64.msi
-        uninstall_flags: '/qn /norestart'
+    .. code-block:: yaml
 
-:param str arch: This selects which ``msiexec.exe`` to use. Possible values:
-    ``x86``, ``x64``
+        7zip:
+          '9.20.00.0':
+            installer: salt://win/repo/7zip/7z920-x64.msi
+            full_name: 7-Zip 9.20 (x64 edition)
+            reboot: False
+            install_flags: '/qn /norestart'
+            msiexec: True
+            uninstaller: salt://win/repo/7zip/7z920-x64.msi
+            uninstall_flags: '/qn /norestart'
+
+:param msiexec:
+    This tells salt to use ``msiexec /i`` to install the package and
+    ``msiexec /x`` to uninstall. This is for ``.msi`` installations. Possible
+    options are: True, False or the path to ``msiexec.exe`` on your system
 
-:param bool allusers: This parameter is specific to `.msi` installations. It
-    tells `msiexec` to install the software for all users. The default is True.
+    .. code-block:: yaml
 
-:param bool cache_dir: If true when installer URL begins with salt://, the
-    entire directory where the installer resides will be recursively cached.
-    This is useful for installers that depend on other files in the same
-    directory for installation.
+        7zip:
+          '9.20.00.0':
+            installer: salt://win/repo/7zip/7z920-x64.msi
+            full_name: 7-Zip 9.20 (x64 edition)
+            reboot: False
+            install_flags: '/qn /norestart'
+            msiexec: 'C:\Windows\System32\msiexec.exe'
+            uninstaller: salt://win/repo/7zip/7z920-x64.msi
+            uninstall_flags: '/qn /norestart'
+
+:param bool allusers:
+    This parameter is specific to ``.msi`` installations. It tells ``msiexec``
+    to install the software for all users. The default is ``True``.
+
+:param bool cache_dir:
+    If ``True`` and the installer URL begins with ``salt://``, the entire
+    directory where the installer resides will be recursively cached. This is
+    useful for installers that depend on other files in the same directory for
+    installation.
+
+    .. warning::
+        Do not place installer files in the root of winrepo
+        (``/srv/salt/win/repo-ng``). If the installer is in the root of winrepo
+        and the package definition for that installer has ``cache_dir: True``
+        then the entire contents of winrepo will be cached to the minion.
```

Comment:
@morganwillcock Here's the rest of the warning

Reply:
It isn't just at the root level this will happen, because .sls files can be in sub-folders (all of mine are, but installers are in that same sub-folder).